public/html/index.html

```diff
@@ -1,5 +1,10 @@
 <!DOCTYPE html>
 <html class="mode__full-page light">
+  <!--
+    Hello! Thank you for showing interest in our source code.
+    Perhaps you'd like to join our team? To learn more, email us:
+    wallet@hiro.so
```

Comment:
I'd use slightly less formal language "Thanks" etc 

Suggest 

```suggestion
    Hello! Thanks for showing interest in our code.
    Interested in joining our team? To learn more, email us:
    wallet@hiro.so
```